docs/team/Sheikh-Umar.adoc

```diff
@@ -21,10 +21,8 @@ Hence, this enhancement tackles this issues so that the user will have a more pl
 ** Highlights: This enhancement affects existing commands and commands to be added in future. It required an in-depth analysis of design alternatives.
 
 * *Minor enhancement*: Created an alias command for basic commands
-** What it does: User can make the CRM Book perform basic commands with a few letters.
-** Justification: Salespeople may want the CRM Book to do a basic command as quickly as possible. This will involve typing as few letters as possible.
```

Comment:
why did you remove the justification and highlights?

Reply:
I thought I did not need to elaborate the minor enhancement as much as the major enhancement. Will re-add it shortly.